### PR TITLE
[RTM] Only output width and height attributes if they are set

### PR DESCRIPTION
**Pull request by ausi**
See #1163

### CODE REVIEW

File: src/Resources/contao/templates/picture/picture_default.html5 [hunk 1]
```diff
@@ -8,7 +8,7 @@
...
+<img src="<?= $this->img['src'] ?>"<?php if ($this->img['srcset'] !== $this->img['src']) echo ' srcset="' . $this->img['srcset'] . '"'; ?><?php if (!empty($this->img['sizes'])) echo ' sizes="' . $this->img['sizes'] . '"'; elseif (!$this->sources && $this->img['width'] && $this->img['height']) echo ' width="' . $this->img['width'] . '" height="' . $this->img['height'] . '"'; ?> alt="<?= $this->alt ?>"<?php if ($this->title) echo ' title="' . $this->title . '"'; ?><?php if ($this->class || !empty($this->img['class'])) echo ' class="' . trim($this->class. ' ' . $this->img['class']) . '"'; ?><?= $this->attributes ?> itemprop="image">
```
Comment (leofeyer) [inline src/Resources/contao/templates/picture/picture_default.html5]: Shouldn't this be `!empty($this->img['width']) && !empty($this->img['height'])`?

Comment (ausi) [reply src/Resources/contao/templates/picture/picture_default.html5]: Done in 336603e2a497e62dc9de387d2b8717d6a612856b

**Comment by ausi** [comment]
@leofeyer should we backport this to Contao 3.5?

**Comment by leofeyer** [comment]
No. It is May 2018 so Contao 3.5 is in the "security fixes only" phase of the long term support. And I don't want to risk any core developer leaving the team. 😉

**Comment by ausi** [comment]
Ah, I missed that 3.5 ist out of LTS now. No backport needed then 👍

**Pull request merged**